pkg/channel/consolidated/reconciler/dispatcher/kafkachannel.go

```diff
@@ -127,7 +127,16 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 	kafkaChannelInformer.Informer().AddEventHandler(
 		cache.FilteringResourceEventHandler{
 			FilterFunc: filterWithAnnotation(injection.HasNamespaceScope(ctx)),
-			Handler:    controller.HandleAll(r.impl.Enqueue),
+			Handler: cache.ResourceEventHandlerFuncs{
+				AddFunc:    r.impl.Enqueue,
+				UpdateFunc: controller.PassNew(r.impl.Enqueue),
+				DeleteFunc: func(obj interface{}) {
+					// TODO when finalize kind is fixed, we'll need to handle that error properly
```

Comment:
what's the fix (or issue for the `FinalizeKind`) ?

Reply:
ah, I see: https://github.com/knative/pkg/issues/2021

thanks

Reply:
https://github.com/knative-sandbox/eventing-kafka/pull/509#discussion_r608467643